bucket/ironwail.json

```diff
@@ -0,0 +1,87 @@
+{
+    "version": "0.6.0",
+    "description": "A fork of the popular GLQuake descendant QuakeSpasm with a focus on high performance instead of maximum compatibility, with a few extra features sprinkled on top.",
```

Comment:
```suggestion
    "description": "Fork of QuakeSpasm with a focus on high performance instead of maximum compatibility, with a few extra features sprinkled on top",
```